app/index.jsx

```diff
@@ -22,15 +22,17 @@ $("#collapseButton").on("click", function() {
 
   if (dataP.hasClass("panel-collapsed") && templateV.hasClass("panel-collapsed")) {
 
-    dataP.animate({ "width": "20%"}, animationDuration, animationEasing, setCollapse(dataP, false));
-    templateV.animate({ width: "20%"}, animationDuration, animationEasing, setCollapse(templateV, false));
+    dataP.animate({ "width": "20%", opacity: "1.0"}, animationDuration, animationEasing, setCollapse(dataP, false));
+    templateV.animate({ width: "20%", opacity: "1.0"}, animationDuration, animationEasing, setCollapse(templateV, false));
     visualizerW.animate({width: "60%"});
+    $("#collapseButton").text("Collapse");
 
     } else {
 
-    dataP.animate({ width: "0%"}, animationDuration, animationEasing, setCollapse(dataP, true));
-    templateV.animate({ width: "0%"}, animationDuration, animationEasing, setCollapse(templateV, true));
+    dataP.animate({ width: "0%", opacity: "0.0"}, animationDuration, animationEasing, setCollapse(dataP, true));
+    templateV.animate({ width: "0%", opacity: "0.0"}, animationDuration, animationEasing, setCollapse(templateV, true));
     visualizerW.animate({width: "100%"});
+    $("#collapseButton").text("Expand");
```

Comment:
let's stick with a naming convention on CSS classes/IDs of `collapse-button`, having a cconsistent rule for this reduces cognitive overhead of having to think about which case was used. i.e. when i am editing css i want to just memorize the words and apply the rule, not remember/forget that is was camelCase at one point and snake-case at another.